File: bin/static/src/app/issuing-module/shared/model/configuration/input-properties.ts
import { JsonMember, JsonObject } from 'typedjson-npm/src/typed-json';

import { VariableDataOption } from '../configuration/variable-data-option';

/**
* this object represents a input properties for variable data
*
* @author jtorres
*/
@JsonObject
export class InputProperties {

    @JsonMember
    label: string;
    @JsonMember
    typeVariable: string;
    @JsonMember
    multiple: boolean;
    @JsonMember({elements: VariableDataOption})
    optionList: VariableDataOption[];
}

File: src/main/java/co/com/fsighttech/imp/issuing/bo/configuration/InputProperties.java
/**
 * 
 */
package co.com.fsighttech.imp.issuing.bo.configuration;

import java.util.List;

import lombok.Getter;
import lombok.Setter;

/**
 * This class represents an input properties for a variable data
 * 
 * @author jtorres
 *
 */
@Getter
@Setter
public class InputProperties {

	private String label;
	private String typeVariable;//(radio button (RB), checkbox list (CL), open (OP) and selection list (SL))
	private boolean multiple;
	private List<VariableDataOption> optionList;
}
